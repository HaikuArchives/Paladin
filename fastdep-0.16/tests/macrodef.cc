#define BUFSIZE 1001

#if BUFSIZE==1001
#include "a.h"
#endif

#if BUFSIZE==1000
#include "b.h"
#endif
