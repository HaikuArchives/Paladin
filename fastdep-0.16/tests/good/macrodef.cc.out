macrodef.o: macrodef.cc \
	a.h \
	c.h \
	d.h
